### PR TITLE
Add a simple test

### PR DESCRIPTION
**Pull request by kamahen**

### CODE REVIEW

File: test/test_hdt.pl [hunk 1]
```diff
@@ -0,0 +1,41 @@
...
+   See the comments in data/README.md if a test fails - the failure
+   might be because the data has changed.
+*/
+
+:- module(test_hdt,
+         [ test_hdt/0
+         ]).
+
+:- use_module(library(plunit)).
+:- use_module(library(hdt)).
```
Comment (kamahen) [inline test/test_hdt.pl:13]: Is this the correct way to get the pack that has just been built?

File: test/test_hdt.pl [hunk 2]
```diff
@@ -0,0 +1,41 @@
...
+                       1:1:6,
+                       1:1:7,
+                       1:1:8,
+                       1:1:9]),
+    hdt_close(H).
+
+hdt_open_literals(H) :-
+    predicate_property(test_hdt, file(TestHdtFile)),
+    directory_file_path(TestHdtDir, _, TestHdtFile),
+    atom_concat(TestHdtDir, '/../hdt-cpp/libhdt/data/literals.hdt', LiteralsHdtPath),
```
Comment (kamahen) [inline test/test_hdt.pl:37]: Is this the best way to pick up a file relative to the directory that test_hdt.pl is in?

Comment (JanWielemaker) [reply test/test_hdt.pl:37]: That is a clean and safe way to find files relative to the current file.

**Comment by JanWielemaker** [comment]
Good start.    We may want to add our own data, both to control it and to test all literal representations.

**Pull request merged**